lib/graph3d/Graph3d.js

```diff
@@ -1680,6 +1646,40 @@ Graph3d.prototype._getColorsSize = function() {
 };
 
 
+/**
+ * Determine the color corresponding to a given value on the color scale.
+ *
+ * @param {number} [x] the data value to be mapped running from 0 to 1
+ * @param {number} [v] scale factor between 0 and 1 for the color brightness
+ * @returns {string}
+ * @private
+ */
+Graph3d.prototype._colormap = function(x, v) {
+  if (v === undefined) {
+    v = 1;
+  }
+
+  var colormap = this.colormap;
+  if (Array.isArray(colormap)) {
+    var maxIndex = colormap.length - 1;
+    var startIndex = Math.max(Math.floor(x * maxIndex), 0);
+    var endIndex = Math.min(startIndex + 1, maxIndex);
+    var innerRatio = x * maxIndex - startIndex;
+    var min = colormap[startIndex];
+    var max = colormap[endIndex];
+    var r = min.r + innerRatio*(max.r - min.r);
+    var g = min.g + innerRatio*(max.g - min.g);
+    var b = min.b + innerRatio*(max.b - min.b);
+  } else if (typeof colormap === 'function') {
+    var {r, g, b} = colormap(x);
+  } else {
+    var hue = (1 - x) * 240;
+    var {r, g, b} = util.HSVToRGB(hue/360, 1, 1);
+  }
+  return `RGB(${parseInt(r*v)}, ${parseInt(g*v)}, ${parseInt(b*v)})`;
```

Comment:
What's the point of parseInt on number?

Reply:
Turn float to integer because the css `rgb()`[https://www.w3schools.com/cssref/func_rgb.asp] function wants integers. I've copied it from `visjs.util`.

Reply:
Copied from where? To me this seems like a weird and confusing alternative to `Math.floor(…)`.

Reply:
Carried over from the old [Graph3d._hsv2rgb](https://github.com/visjs/vis-graph3d/blob/master/lib/graph3d/Graph3d.js#L1440) and also seen in [vis.util](https://github.com/visjs/vis-util/blob/master/src/util.ts#L916). But you're right, I will change to Math.round, I think that makes the most sense and is also what is used in chroma.js.